server.go

```diff
@@ -490,6 +521,21 @@ func (s *server) Peers() []*peer {
 	return <-resp
 }
 
+// Channels returns a slice of all channels.
+func (s *server) Channels() ([]*channeldb.OpenChannel, error) {
+	respChan := make(chan []*channeldb.OpenChannel)
```

Comment:
Since access to the database is already serialized, `channeldb.FetchAllChannels()` can be accessed directly.